dcc-portal-ui/app/scripts/repository/views/repository.icgc.html

```diff
@@ -76,8 +76,8 @@ <h1 data-ui-scrollfix=79>
                <div class="well well-small">
                    <div><a target="_self" href="{{ICGCRepoController.API.BASE_URL}}/download?fn={{file.name}}"><i class="icon-file"></i>{{file.baseName}}</a></div>
                    <hr>
-                   <div class="markdown_container">
-                       <div btf-markdown="file.textContent"></div>
+                   <div class="markdown_container" >
+                       <div btf-markdown="file.textContent" class="markdown_wrapper"></div>
```

Comment:
I realize you were just following suit here, but can we change the classes to be `markdown-wrapper` and `markdown_container`? Make sure this is synchronized in other files (directive, stylesheets).